gradle-examples/README.md

```diff
@@ -68,4 +68,6 @@ The first time you should build this project with:
 After downloading the correct Gradle version, it will take about about 11s and push the cache to Artifactory. During 
 tests execution, you should see the message `Executing heavy fake test`. 
 Then if you try it from a different environment, or simply rebuild with `./gradlew clean build`, 
-it will skip the test task, fetch the cache from Artifactory instead and should take about 1s.
\ No newline at end of file
+it will skip the test task, fetch the cache from Artifactory instead and should take about 1s.
+
+Read more information about [Gradle repository](https://jfrog.com/integration/gradle-repository/) and [Gradle Artifactory plugin](https://www.jfrog.com/confluence/display/RTF/Gradle+Artifactory+Plugin).
```

Comment:
Read more information about [Artifactory as a Gradle repository](https://jfrog.com/integration/gradle-repository/) and the [Gradle Artifactory Plugin](https://www.jfrog.com/confluence/display/RTF/Gradle+Artifactory+Plugin).
Also read read about [JFrog CLI's integration with Gradle and Maven](https://www.jfrog.com/confluence/display/CLI/CLI+for+JFrog+Artifactory#CLIforJFrogArtifactory-RunningMavenandGradleBuilds)